opencsp/app/sofast/lib/MeasurementSofastFixed.py

```diff
@@ -33,20 +34,19 @@ def __init__(
             Optic to screen distance, meters
         origin : Vxy
             The centroid of the origin dot, pixels
-        date : datetime
-            Collection date/time.
-        name : str
-            Name or serial number of measurement.
+        date : datetime, optional
+            Collection date/time. Default is dt.datetime.now()
+        name : str, optional
+            Name or serial number of measurement. Default is empty string ''
         """
+        super().__init__(dist_optic_screen_measure, date, name)
         self.image = image
-        self.v_measure_point_facet = v_measure_point_facet
-        self.dist_optic_screen = dist_optic_screen
         self.origin = origin
         self.date = date
         self.name = name
```

Comment:
Do these need to be here (line 45/46) since they're already set in super().__init__()?

Reply:
They sure don't. Good catch again!